gulp/build.js

```diff
@@ -46,7 +46,11 @@ gulp.task('html', ['inject', 'partials'], function () {
     .pipe($.uglify({preserveComments: $.uglifySaveLicense}))
     .pipe(jsFilter.restore())
     .pipe(cssFilter)
-    .pipe($.replace('../../bower_components/fontawesome/fonts', '../fonts'))
+    //.pipe($.replace('../../bower_components/fontawesome/fonts', '../fonts'))
+    .pipe($.base64({
```

Comment:
For iPython, we cannot load pictures or fonts, so I have to embed them with base64 encoding into the CSS. It actually reduces the number of requests, which is nice, but increases the overall data size. 

I was too lazy to add a build flag to build different versions.